components/Testimonial.vue

```diff
@@ -1,21 +1,118 @@
 <template>
-  <div>
-    <Tweet :id="tweetId" :options="{ cards: 'hidden' }" />
-  </div>
+  <v-card
+    class="mx-auto tweet-widget"
+    :class="$vuetify.breakpoint.mobile ? 'mobile' : ''"
+    color="white"
+  >
+    <v-carousel
+      v-model="tweetIndex"
+      hide-delimiters
+      :show-arrows="!$vuetify.breakpoint.mobile"
+      light
+    >
+      <v-carousel-item v-for="(tweet, i) in tweets" :key="tweet.id">
+        <tweet :tweet="tweets[i]" />
+      </v-carousel-item>
+    </v-carousel>
+    <tweet-carousel-delimiter
+      v-show="$vuetify.breakpoint.mobile"
+      :steps="3"
+      :index="tweetIndex"
+    />
+  </v-card>
 </template>
-
 <script>
-import { Tweet } from "vue-tweet-embed";
-import json from "../data/testimonials.json";
+import testimonialsData from "../data/testimonials.json";
+import tweetFixtures from "../fixtures/twitterTweetsApiFixture.json";
+
+const API_URL =
+  "https://api.twitter.com/2/tweets?user.fields=profile_image_url,username&expansions=author_id&ids=";
+const DEV_MODE = "DEV_MODE";
 
-const idx = Math.floor(Math.random() * json.tweets.length);
+const token = process.env.TWITTER_API_BEARER_TOKEN || DEV_MODE;
+const options = {
+  headers: { Authorization: `Bearer ${token}` },
+  mode: "no-cors",
+};
 
 export default {
-  components: { Tweet },
   data() {
     return {
-      tweetId: json.tweets[idx],
+      tweets: [],
+      tweetIndex: 0,
```

Comment:
perhaps we should rename this to be more clear on what the index is for, like `activeTweetIndex`.

Reply:
sure, no strong opinion. happy to update.